services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py

```diff
@@ -2,13 +2,15 @@
 from typing import Any, Optional
 
 from models_library.basic_types import BootModeEnum, PortInt
+from models_library.settings.docker_registry import RegistrySettings
 from pydantic import BaseSettings, Field, PositiveInt, validator
 
 
 class DynamicSidecarSettings(BaseSettings):
```

Comment:
Please read carefully this [PR](https://github.com/ITISFoundation/osparc-simcore/pull/2395) about how the settings work now.